app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx

```diff
@@ -0,0 +1,154 @@
+  }
+}
+
+export class MultiSelectCheckbox extends React.Component<IServerGroupHeaderProps> {
+  public render() {
+    const { isMultiSelected, sortFilter: { multiselect } } = this.props;
+    return multiselect && <input type="checkbox" checked={isMultiSelected}/>;
+  }
+}
+
+export class CloudProviderIcon extends React.Component<IServerGroupHeaderProps> {
+  public render() {
+    const { serverGroup } = this.props;
+    return <CloudProviderLogo provider={serverGroup.type} height="16px" width="16px"/>;
+  }
+}
+
+export class Sequence extends React.Component<IServerGroupHeaderProps> {
+  public render() {
+    const { serverGroup } = this.props;
+    const serverGroupSequence = ReactInjector.namingService.getSequence(serverGroup.moniker.sequence);
+    return <span className="server-group-sequence"> {serverGroupSequence}</span>;
+  }
+}
+
+export class BuildAndImages extends React.Component<IServerGroupHeaderProps> {
+  public render() {
+    const { jenkins, images } = this.props;
+    return (
+      <div>
+        {(!!jenkins || !!images) && <span>: </span>}
+        {!!jenkins && <a className="build-link" href={jenkins.href} target="_blank">Build: #{jenkins.number}</a>}
+        {!!images && <span>{images}</span>}
+      </div>
+    );
+  };
+}
+
+export class Alerts extends React.Component<IServerGroupHeaderProps> {
+  public render() {
+    const { application, serverGroup } = this.props;
+    return (
+      <EntityNotifications
+        application={application}
+        entity={serverGroup}
+        entityType="serverGroup"
+        hOffsetPercent="20%"
+        onUpdate={() => application.serverGroups.refresh()}
+        pageLocation="pod"
+        placement="top"
+      />
+    );
+  };
+}
+
+@Overridable('serverGroups.pod.header.health')
+export class Health extends React.Component<IServerGroupHeaderProps> {
+  public render() {
+    const { serverGroup } = this.props;
+    return <HealthCounts className="no-float" container={serverGroup.instanceCounts}/>;
+  }
+}
+
+export class RunningTasks extends React.Component<IServerGroupHeaderProps> {
+  public render() {
+    const { application, serverGroup } = this.props;
+    const { RunningTasksTag } = NgReact;
+    const hasRunningExecutions = !!serverGroup.runningExecutions.length || !!serverGroup.runningTasks.length;
+
+    return hasRunningExecutions && (
+      <RunningTasksTag
+        application={application}
+        tasks={serverGroup.runningTasks}
+        executions={serverGroup.runningExecutions}
+      />
+    );
+  };
+}
+
+export class ServerGroupManager extends React.Component<IServerGroupHeaderProps> {
+  public render() {
+    const { application, serverGroup } = this.props;
+    return <ServerGroupManagerTag application={application} serverGroup={serverGroup}/>;
+  }
+}
+
+@Overridable('serverGroups.pod.header')
+export class ServerGroupHeader extends React.Component<IServerGroupHeaderProps> {
+  public headerIsSticky(): boolean {
```

Comment:
I wonder if we still need this, or if we should always set the headers to be sticky, even if there's just one instance showing. I think historically we limited it because we were doing a lot of measuring and calculation in JS, but we're not doing that with the `position: sticky` approach now.

I played around with it always enabled on a larger, mixed size application, and it felt fine to me.

Reply:
Sure, seems fine to always sticky. I deleted the conditional.